WalletWasabi.Fluent/Models/WebcamQrReader.cs

```diff
@@ -9,162 +9,81 @@
 using NBitcoin;
 using Nito.AsyncEx;
 using Avalonia.Platform;
+using WalletWasabi.Fluent.Models.Windows;
+using Avalonia.Controls;
+using Microsoft.Extensions.Hosting;
+using System.Threading;
+using ZXing.QrCode;
+using WalletWasabi.Bases;
 
 namespace WalletWasabi.Fluent.Models;
 
-public class WebcamQrReader
+public class WebcamQrReader : PeriodicRunner
```

Comment:
I wonder, why?

Reply:
Because the MainThread will be overwhelmed if we send back a new picture from the camera every tick.
Instead, we only send back a picture once every 100 miliseconds, and for this task a PeriodicRunner is the sollution.